test/string_test.rb

```diff
@@ -273,6 +273,29 @@
     end
   end
 
+  describe '#singularize' do
+    before do
+      @singular, @plural = *TEST_SINGULARS.to_a.sample
```

Comment:
@rogeriozambon @joneslee85 Better random values for testing. We want to avoid to design tests around well known values.